doc/plutus-core-spec/cardano/builtins1.tex

```diff
@@ -57,31 +57,25 @@ \subsubsection{Built-in types and type operators}
 Unicode replacement character \texttt{U+FFFD}.
 
 
-\paragraph{Concrete syntax for higher-order types.} Types such as $\listOf{\ty{integer}}$
-and $\pairOf{\ty{bool}}{\ty{string)}}$ are represented by application at the
-type level, thus: \texttt{[(con list) (con integer)]} and\texttt{[(con pair)
-    (con bool) (con string)]}.  Each higher-order type will need further syntax
-for representing constants of those types.  For example, we might use
-\texttt{[]} for list values and \texttt{(,)} for pairs, so the list $[11,22,33]$
-might be written as
-\begin{verbatim}
-   (con [(con list) (con integer)] 
-        [(con integer 11), (con integer 22), (con integer 33)]
-   )
-\end{verbatim}
-and the pair (True, "Plutus") as
+\paragraph{Concrete syntax for lists and pairs.}
+A list of type $\texttt{list}(t)$ is written as a syntactic list
+\texttt{[$c_1, \ldots, c_n$]} where each $c_i$ lies in $\bitc_t$; a pair of type
+$\texttt{pair}(t_1,t_2)$ is written as a syntactic pair $\texttt{(}c_1,c_2\texttt{)}$
+with $c_1 \in \bitc_{t_1}$ and $c_2 \in \bitc_{t_2}$.  Some valid constant expressions
+are thus
+
 \begin{verbatim}
-   (con [(con pair) (con bool) (con string)] 
-        ((con bool True), (con string "Plutus"))
-   ).
+   (con (list integer) [11, 22, 33])
+   (con (pair bool string) (True, "Plutus")).
+   (con (list (pair bool (list bytestring)))
+      [(True, []), (False, [#,#1F]), (True, [#123456, #AB, #ef2804])])
```

Comment:
Can a bytestring be `#`? I thought so, but according to Section 4.3.1.1, the bytestring syntax is `#([0-9A-Fa-f][0-9A-Fa-f])*`, which must be non-empty.

Reply:
`*` means zero or more repetitions, so that should be OK.

Reply:
OK, I misread it.